Sources/ClientRuntime/Networking/Http/CRT/HTTP2Stream+ByteStream.swift

```diff
@@ -19,13 +20,24 @@ extension HTTP2Stream {
     /// There is no recommended size for the data to write. The data will be written in chunks of `manualWriteBufferSize` bytes.
     /// - Parameter body: The body to write
     /// - Throws: Throws an error if the write fails
-    func write(body: ByteStream) async throws {
+    func write(body: ByteStream, telemetry: HttpTelemetry, serverAddress: String) async throws {
```

Comment:
Plug telemetry when writing an `HTTP2Stream`. This could also be extracted out of the extension and be inlined, which would prevent `telemetry` and `serverAddress` polution